backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarPartialProjectExec.scala

```diff
@@ -141,21 +136,20 @@ case class ColumnarPartialProjectExec(
     if (projectAttributes.size == child.output.size) {
       return ValidationResult.failed("UDF need all the columns in child output")
     }
-    if (original.output.isEmpty) {
-      return ValidationResult.failed("Project fallback because output is empty")
-    }
```

Comment:
If `original.output` is empty, then `replacedAliasUdf` must be also empty, repeat the following judgment.